### PR TITLE
ibu: add extra check for stage to avoid going into poll

### PR DESCRIPTION
**Pull request by achuzhoy**

### CODE REVIEW

File: pkg/lca/imagebasedupgrade.go [hunk 1]
```diff
@@ -381,6 +384,10 @@ func (builder *ImageBasedUpgradeBuilder) WaitUntilStageComplete(stage string) (*
 		return builder, fmt.Errorf(builder.errorMsg)
 	}
 
+	if stage != idle && stage != "Prep" && stage != "Upgrade" && stage != "Rollback" {
```
Comment (trewest) [inline pkg/lca/imagebasedupgrade.go:387]:
```suggestion
	if !slices.Contains([]string{"Idle", "Prep", "Upgrade", "Rollback"}, stage) {
```

Comment (achuzhoy) [reply pkg/lca/imagebasedupgrade.go]: adjusted. Thanks.

**Review by trewest — APPROVED** [review]
/lgtm

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**